projects/amulet-protocol/index.js

```diff
@@ -19,10 +19,22 @@ async function tvl() {
     const accountSolBalances = await Promise.all(accountList.map(async(x) => new BN((await connection.getAccountInfo(new PublicKey(x))).lamports)))
     const solAmount = [new BN(posSolVirtualAmount), new BN(liqSolAmount), new BN(totalActiveBalance), ...accountSolBalances]
 
-    const tvl = solAmount.reduce((acc,x) => acc.add(x), new BN(0)).div(new BN(1000000000)).toNumber()
+    const tvlSOL = solAmount.reduce((acc,x) => acc.add(x), new BN(0)).div(new BN(1000000000)).toNumber()
```

Comment:
can you remove this? it is already counted in projects/amulet-liquidity-staking

Reply:
Sure, will update and re-submit

Reply:
Updated and re-submitted.